README.md

```diff
@@ -42,7 +42,7 @@ You can change default JDK version by replacing directory `jre` in dbeaver insta
 - Basic frameworks:
   - [OSGI](https://en.wikipedia.org/wiki/OSGi) platform for plugins and dependency management. Community version consists of 130+ plugins.
   - [Eclipse RCP](https://github.com/eclipse-platform/eclipse.platform.ui/blob/master/docs/Rich_Client_Platform.md) platform for rich user interface build.
-  - [JDBC](https://en.wikipedia.org/wiki/Java_Database_Connectivity) for basic databases connectivity API.
+  - [JDBC](https://en.wikipedia.org/wiki/Java_Database_Connectivity) for basic database connectivity API.
```

Comment:
"basic database**s** connectivity" - the databases are not basic, the connectivity is, therefore drop plural